astropy/cosmology/tests/test_cosmology.py

```diff
@@ -1088,13 +1088,13 @@ def test_critical_density():
     #  units by default; see the comment at the top of core.py
     tcos = core.FlatLambdaCDM(70.4, 0.272, Tcmb0=0.0)
     assert allclose(tcos.critical_density0,
-                    9.309668456020899e-30 * u.g / u.cm**3)
+                    9.309361588325364e-30 * u.g / u.cm**3)
```

Comment:
I think I'd prefer to just add a larger `atol` with a comment that this is because of uncertainty in `G` (and thus changes in its value between constant versions). But also fine to leave as is.

Reply:
What value of `atol` is good for this? I didn't study `cosmology` enough to know what `atol` is acceptable for such a test.

Reply:
I guess it should really be an `rtol`, equal to the uncertainty in `G`. 1e-4 would cover it. Alternatively, multiply with `const.G.value / (value of G from codata2014)`